sam-styles/packages/components/button/styles/button.scss

```diff
@@ -514,8 +515,8 @@ $button-shadow: 0 0.25rem 0.5rem 0 rgba(0, 0, 0, 0.2);
   &--disabled,
   &:disabled {
     box-shadow: none;
-    @include u-border("disabled");
-    @include u-bg("disabled");
+    @include u-border("base-light");
```

Comment:
According to the state colors on the [dev site](https://gsa.github.io/sam-styles/?path=/story/branding-colors--state-colors) disabled and base-light should be the same color. Are they not actually the same color?

If they are the same, wouldn't we want to keep applying the "disabled" color to the disabled element, in case that color ever needed to change from being the same as base-light?

Reply:
so I found the reason causing this issue. uswds changed their disabled setting color to base and we didnt have a token for disabled on our end and were just using what uswds had, but because they changed their disabled color, we had to add a token to use our disabled color (base-light / gray-20). Which I updated in the latest commit.

<img width="634" alt="Screen Shot 2023-10-18 at 1 48 53 PM" src="https://github.com/GSA/sam-styles/assets/118462746/b26fb995-71ef-47c1-a3f8-04b23e22d600">